File: LogicaDeProgramacaoEclipse(Java)/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/exerciciosFaccat/Exercicio16.java
package exerciciosFaccat;

import java.util.Scanner;

/*16) As maçãs custam R$ 1,30 cada se forem compradas menos de uma dúzia, e R$ 1,00 se forem
compradas pelo menos 12. Escreva um programa que leia o número de maçãs compradas, calcule e
escreva o custo total da compra. */

public class Exercicio16 {
	public static void main(String []args) {
		Scanner sc = new Scanner(System.in);
		
		double soma;

		System.out.println("Digite a quantidade de Maçãs que deseja comprar");
		int quantidadeMacas = sc.nextInt();

		if (quantidadeMacas >= 12) {
			soma = quantidadeMacas * 1;
			System.out.println("Como foi pedido " + quantidadeMacas + " Maçãs, o preço fica R$" + soma + ",00");
		} else {	
		   soma =  1.3 * quantidadeMacas;
		  System.out.println("Como foi pedido " + quantidadeMacas + " Maçãs, o preço fica R$" + soma);
		}
		
		sc.close();
	}
}

File: LogicaDeProgramacao(JavaScript)/Exercicios 5 até o 39 Apostila Faccat/js/exercicio16.js
/*16) As maçãs custam R$ 1,30 cada se forem compradas menos de uma dúzia, e R$ 1,00 se forem
compradas pelo menos 12. Escreva um programa que leia o número de maçãs compradas, calcule e
escreva o custo total da compra. */

alert("16) As maçãs custam R$ 1,30 cada se forem compradas menos de uma dúzia, e R$ 1,00 se forem compradas pelo menos 12. Escreva um programa que leia o número de maçãs compradas, calcule e escreva o custo total da compra")

var quantidadeMacas = parseInt(prompt("Digite o valor da quantidade de Maçãs que você quer:"))
var valorMacas = 0
if (quantidadeMacas < 12) {
    valorMacas = 1.30 * quantidadeMacas
    alert("O valor de " + quantidadeMacas + " Maçãs é: " + valorMacas)
} else if (quantidadeMacas >= 12) {
    valorMacas = 1 * quantidadeMacas
    alert("O valor de " + quantidadeMacas + " Maças é: " + valorMacas)
}